Commons/src/main/java/eu/neclab/ngsildbroker/commons/datatypes/Subscription.java

```diff
@@ -167,6 +183,9 @@ public static Subscription expandSubscription(Map<String, Object> body, String i
 						case NGSIConstants.JSON_LD_TYPE:
 							hasType = true;
 							entityInfo.setType(((List<String>) entitiesEntry.getValue()).get(0));
+							if(entityInfo.getType().equals("https://uri.etsi.org/ngsi-ld/default-context/*")){
```

Comment:
Add in NGSIConstants "https://uri.etsi.org/ngsi-ld/default-context/*"

Reply:
done